client/src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import ReviewsListEntity from './ReviewsListEntity.jsx';
+import ReviewsPages from './ReviewsPages.jsx';
+
+class ReviewsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+        ordered_reviews: this.props.reviews,
+        rendering_order: 'Most recent',
+        page_num: 1
+    }
+
+    this.changeReviewsOrder = this.changeReviewsOrder.bind(this);
+    this.changeReviewsPage = this.changeReviewsPage.bind(this);
+  }
+
+  changeReviewsOrder(event) {
+      // sort reviews according to selected order
+      let order = event.target.value;
+      let ordered_reviews = [];
+
+      if (order === 'Most recent') {
+        ordered_reviews = this.props.reviews.sort( (a,b) => {
+            return new Date(b.update_date) - new Date(a.update_date);
+        });
+
```

Comment:
I think it would be better to delete this blank line in between the `if` block and the `else if` block so that it's easier to tell that they are part of the same statement.

Reply:
👍